<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BufferCompositionWidget</class>
 <widget class="QWidget" name="BufferCompositionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CopiableItemsTableView" name="qtbv_composition"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="qpb_add">
       <property name="text">
        <string>Add BGE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_remove">
       <property name="text">
        <string>Remove BGE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopiableItemsTableView</class>
   <extends>QTableView</extends>
   <header>src/ui/copiableitemstableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
